platform-includes/distributed-tracing/custom-instrumentation/browser/javascript.mdx

```diff
@@ -0,0 +1,100 @@
+
+Distributed tracing will be set up automatically if you've <PlatformLink to="/tracing/">set up tracing</PlatformLink>.
+
+When tracing is enabled, the Sentry SDK will not only create spans for pageloads and transactions but also propagate trace information to other systems. This allows you to connect multiple systems and see how they interact with each other.
+
+If you have not enabled tracing, or you need to propagate traces in places other than outgoing HTTP requests (e.g. when working with websockers), you can manually set up your application for distributed tracing to work.
+
+## Enabling Automatic Distributed Tracing
+
+To enable distributed tracing for your frontend, add `browserTracingIntegration` to your `Sentry.init()` options as described in the <PlatformLink to="/tracing/instrumentation/automatic-instrumentation/">Automatic Instrumentation</PlatformLink> docs.
+
+If you want to use distributed tracing without creating spans, set the `tracesSampleRate` option to `0`.
```

Comment:
Why would I want to do this? Maybe I completely misunderstand the whole concept of tracing... but how can you trace something without creating spans? How does it look in Sentry?

Reply:
We have this page that tries to explain the concept of distributed tracing: https://docs.sentry.io/platforms/javascript/guides/react/tracing/trace-propagation/#what-is-distributed-tracing - is that helpful? It's a bit hard to describe/explain this here again without exceeding the scope of it considerably 😬 I can try to put an expandable in there!

Reply:
Nope sorry, that page doesn't answer why I would want to NOT create spans and still see distributed traces in Sentry. I don't understand.